doc/conf.py

```diff
@@ -102,7 +102,7 @@
 # built documents.
 #
 # The short X.Y version.
-version = xarray.version.short_version
+version = xarray.__version__.split('+')[0]
```

Comment:
this is the one change that worries me. It is possible people were relying on the `xarray.version` module...

Reply:
If they were... then they will have to fix up their code.